qfieldsync/core/cloud_converter.py

```diff
@@ -135,10 +136,10 @@ def convert(self) -> None:  # noqa: C901
 
             # TODO whatcha gonna do if QgsProject::read()/write() fails
             if is_converted:
-                self.project.read(project_path)
+                iface.addProject(project_path)
```

Comment:
@suricactus , I was wondering whether we want to avoid using iface in /core. No strong opinion. 

As we discussed yesterday, +1 to rely on addProject along the way in this process.

Reply:
Hey, @nirvn . I also feel a bit nervous depending on `iface`, but I don't think there are a lot of other options.